src/Formatters/HelpFormatter.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Consolidation\OutputFormatters\Formatters;
+
+use Consolidation\OutputFormatters\Options\FormatterOptions;
+use Consolidation\OutputFormatters\Validate\ValidDataTypesInterface;
+use Consolidation\OutputFormatters\Validate\ValidDataTypesTrait;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableCell;
+
+/**
+ * Format an array into CLI help string.
+ */
+class HelpFormatter implements FormatterInterface
```

Comment:
The other thing I was thinking about is perhaps instead of making this a "help" formatter, it could be something like a structured text formatter. The help data could be structured by the help command into sections with headers, and sections could have text, or tables of different types, etc.

Just a thought.